src/cmds/init.py

```diff
@@ -36,12 +36,12 @@ def help_message():
     print(
         """
         chia init (migrate previous version configuration to current)
-        chia init -c (creates new TLS certificates signed by your CA)
+        chia init -c [directory] (creates new TLS certificates signed by your CA directory)
```

Comment:
"signed by your CA in directory"

Reply:
Updated, will push changes after testing DB copy.

Reply:
That also sounds stilted - just want to not say "your CA directory" - maybe "by your CA found in directory"?